src/System.Management.Automation/engine/MshMemberInfo.cs

```diff
@@ -4075,16 +4087,15 @@ public override void Add(T member, bool preValidated)
                 throw PSTraceSource.NewArgumentNullException("member");
             }
 
-            lock (_members)
+            lock (Members)
```

Comment:
would it be better to save a local reference to Members, instead of calling the Property many times in the same would it be better to save a local reference to the members instead of repeatedly call the `Members` property ?
var member = Members;
lock(members)
{
   ...